03week/planet/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header> <h1>The Daily Planet</h1>
+        <nav>
+          <span><a href="#sports">Sports</a></span>
+          <span><a href="#local">Local</a></span>
+          <span><a href="#Economy">Economy</a></span>
+        </nav>
+</header>
+
+    <main><h3><strong>Giannis Antetokounmpo: The Most Intriguing Point Guard In NBA History</strong></h3>
+        <section>
+
+            <article>
+              <img src="img/bucks.jpg" alt="Giannis layup" class="bucks"><p>Antetokounmpo stands 6' 11", with legs so long opposing coaches constantly complain that he is traveling, until they review the tape. “He’s not,” says Wizards coach Scott Brooks. “It’s just that we’ve never seen somebody with a stride like this.” Among the NBA’s legion of stretchy giants, Kevin Durant is the scorer, Anthony Davis the slasher. Antetokounmpo is the creator, traversing half the court with four Sasquatch steps, surveying traffic like a big rig over smart cars. Durant and Davis try to play point guard. Antetokounmpo actually does it, dropping dimes over and around defenders’ heads, leading the Bucks in every major category; 23.8 points, 8.9 rebounds, 5.9 assists, 2.0 blocks and 2.0 steals. This season he will be the team's first All-Star since Michael Redd in 2004, and before you learn to spell his surname, he will be much more.</p>
```

Comment:
Hey DeMarco, your page looks good and it seems like all the CSS elements work. I can't resize my img in my body. What am I doing wrong?